### PR TITLE
www-client/chromium: add ppc64 glue

### PR DESCRIPTION
**Pull request by gyakovlev**
no keywords, but this allows users do download patches
and use gentoo ebuild to install chromium on ppc64le machines

patch downloader script available at:
https://gist.github.com/gyakovlev/d9dee9a554f5f1ff086b15e6f927ebf7

Signed-off-by: Georgy Yakovlev <gyakovlev@gentoo.org>

### CODE REVIEW

File: www-client/chromium/chromium-81.0.4044.113.ebuild [hunk 1]
```diff
@@ -90,7 +90,7 @@ BDEPEND="
 	${PYTHON_DEPS}
 	>=app-arch/gzip-1.7
 	!arm? (
-		dev-lang/yasm
+		!ppc64? ( dev-lang/yasm )
```
Comment (floppym) [inline www-client/chromium/chromium-81.0.4044.113.ebuild:93]: Let's change this to:
```
amd64? ( dev-lang/yasm )
x86? ( dev-lang/yasm )
```

Comment (gyakovlev) [reply www-client/chromium/chromium-81.0.4044.113.ebuild:93]: yeah current double nesting is ugly =)
isn't it needed on arm64 as well?

Comment (floppym) [reply www-client/chromium/chromium-81.0.4044.113.ebuild:93]: > isn't it needed on arm64 as well?

Maybe? I thought yasm only supported amd64 and x86 output though.

Comment (floppym) [reply www-client/chromium/chromium-81.0.4044.113.ebuild:93]: Based on this code search, I think it is only used for amd64 and x86.

https://source.chromium.org/search?q=yasm_assemble&sq=

Comment (stha09) [reply www-client/chromium/chromium-81.0.4044.113.ebuild:93]: AFAICS its only used for building bundled libvpx or we do something wrong ... But yes, only amd64 and x86 (not sure if x86 builds)

Comment (stha09) [reply www-client/chromium/chromium-81.0.4044.113.ebuild:93]: Based on @floppym 's codesearch yasm is only used for libvpx, ffmpeg (win only), openh264 (unbundled) and libaom (not enabled). So correct dep should be:
`!system-vpx (`
`amd64? (dev-lang/yasm)`
`x86? (dev-lang/yasm)`
`)`

Comment (gyakovlev) [reply www-client/chromium/chromium-81.0.4044.113.ebuild]: pushed corrected dep.

btw, should I update chromium-83 ebuild as well with this glue code?
I don't have patches for 83 so I can't actually test it

Comment (stha09) [reply www-client/chromium/chromium-81.0.4044.113.ebuild]: > pushed corrected dep.
> 
> btw, should I update chromium-83 ebuild as well with this glue code?
> I don't have patches for 83 so I can't actually test it

Sure chromium-83 is masked anyway cause it has runtime crashes.

**Comment by floppym** [comment]
Applied as fe9614edcd327cf455b559c6fb7a57edfacc4b57.

**Pull request closed without merging**